Hustle-Hamster/utils.cpp

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <iostream>
+#include <cctype> 
+#include <sstream>
+#include <vector>
+#include "utils.h"
+
+using namespace std;
+
+void delay(int time) {
```

Comment:
I don't think that we need a delay method, I may have forgotten to delete it, but I never use it. I just use the built in Sleep() method.